net/adns/files/patch-common-make-in.diff

```diff
@@ -0,0 +1,11 @@
+--- common.make.in
++++ common.make.in
+@@ -32,7 +32,7 @@ AUTOLDFLAGS=	@LDFLAGS@
+ WARNS=		@WARNS@ -Wno-unused-value
+ #WERROR=	-Werror
+ 
+-M4=		m4
++M4=@MPPREFIX@/bin/gm4
```

Comment:
Since this is a Makefile variable, it seems like it should be possible to override it from the outside by using `build.args-append M4=${prefix}/bin/gm4` instead of patching but I have not tested that.

Reply:
I've tried repeatedly, and this option is ignored whether the build is running locally or on GitHub Actions until I manually patch the file.

Reply:
Potential culprit of the weird destrooting: https://www.chiark.greenend.org.uk/ucgi/~ianmdlvl/git?p=adns.git;a=patch;h=e1849e2848fa6a35990f9b7759997da81b93dc25